recipe/meta.yaml

```diff
@@ -18,6 +18,8 @@ requirements:
   host:
     - pip
     - python 2.7|>=3.6
+    - setuptools >=44
+    - wheel >=0.29
```

Comment:
This does no harm but isn't necessary for conda-forge. :-)

Reply:
Hmm did not know that.

However I think it is good policy to use the exact same dependencies as upstream when possible, so one does not need to think too much if something is necessary or not. 😁

Reply:
Yes, that's perfectly fine.